### ISSUE TITLE
This program needs access to the screen. Please run with a Framework build of python, and only when you are logged in on the main display of your Mac.

### ISSUE THREAD

**Issue by wolfiex** [marcelstoer/nodemcu-pyflasher#46]
Some information on how these conditions are not met would be useful. As far as I am concerned they are.

**Comment by marcelstoer** [comment]
Some information on this WxPython issue here: https://stackoverflow.com/questions/39989975/wx-cant-run-on-mac

**Comment by sshah254** [comment]
Hi,

I ran into this issue too.

When I was building it on my own from the command line, for example,

python3 -m PyInstaller -n NodeMCU-PyFlasher nodeycu-pyflasher.py

then if I run the executable from the dist directory, then I'll get the above error.

But if I build it with

python3 -m PyInstaller -w -n NodeMCU-PyFlasher nodeycu-pyflasher.py

(notice -w option for --windowed)

then I won't get the above error.

There were more issues but that I have listed in issue #61 (https://github.com/marcelstoer/nodemcu-pyflasher/issues/61)

hth,

SS
